packages/theme-base/src/tokens/components/select.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  width: { value: '100%' },
+  transitionProperty: { value: 'background-color' },
+  transitionDuration: { value: '{time.transition.medium.value}' },
+  //variation
+  flushed: {
+    borderTop: { value: 'none' },
+    borderLeft: { value: 'none' },
+    borderRight: { value: 'none' },
```

Comment:
We should try to use logical border properties (borderInlineStart/End etc) instead of borderLeft/borderRight etc

Reply:
I am curious how to use the `borderInlineStart/End ` in this case? Looks like there is no `border-inline-top`? :(